README.md

```diff
@@ -187,7 +187,10 @@ To turn PostHog off, set the `DISABLE_POSTHOG=true` environment variable and rem
 #### Spinning up Keep with docker-compose
 The easiest way to start with Keep is to run it via docker-compose:
 ```shell
-wget -O docker-compose.yml https://raw.githubusercontent.com/keephq/keep/main/docker-compose.yml
+git clone https://github.com/keephq/keep.git
```

Comment:
what do you think about the following, to spare the user from having to clone the entire repo and ease of use:

```suggestion
wget -O docker-compose.common.yml https://raw.githubusercontent.com/keephq/keep/main/docker-compose.yml && wget -O docker-compose.yml https://raw.githubusercontent.com/keephq/keep/main/docker-compose.yml && mkdir -d 0777 -p state
```

let me know

Reply:
Another interesting way might be just downloading some bash script that does all of this for you? (./start-keep.sh) or something, wdyt?